ttnn/cpp/ttnn/operations/uniform/device/kernels/compute_uniform.cpp

```diff
@@ -23,7 +23,7 @@ void MAIN {
     const uint32_t num_tiles = get_arg_val<uint32_t>(4);
     const uint32_t end_id = start_id + num_tiles;
 
-    init_sfpu(intermed_cb_id);
+    init_sfpu(intermed_cb_id, intermed_cb_id);
```

Comment:
this is input/output cb ids?

Reply:
It was done to prevent the default of 16 from being set.   Ideally we would delete all defaults like this but then the PR would get huge.
```
ALWI void init_sfpu(uint32_t icb, uint32_t ocb = 16) {
    unary_op_init_common(icb, ocb);
}
```